src/screens/QRPresentationScreen/index.tsx

```diff
@@ -0,0 +1,53 @@
+import {FC, useEffect, useState} from 'react';
+import {Dimensions} from 'react-native';
+import {SSIBasicContainerSecondaryStyled as SSIContainer} from '../../styles/components';
+import {QRContainer, QRPresentationViewContainer} from 'src/styles/components/screens/QRPresentationScreen';
+import {CreateElementArgs, QRType} from '@sphereon/ssi-sdk.qr-code-generator';
+import {agentContext} from '../../agent';
+
+const {width} = Dimensions.get('screen');
+
+type QRPresentationScreenProps = {};
+
+/* FIXME: Replace this with some result once back end for holder
+ * presentations is complete.
+ */
+const mockURI = 'some_uri_content';
+
+const QRPresentationScreen: FC<QRPresentationScreenProps> = (props: QRPresentationScreenProps) => {
+  const [qrElement, setQrElement] = useState<React.ReactElement | null>(null);
```

Comment:
please import React instead of referencing it

Reply:
I ended up directly importing `ReactElement`